proposals/lip-0046.md

```diff
@@ -180,12 +180,12 @@ The Random module exposes the following logic to other modules.
 
 #### isSeedRevealValid
 
-This function assesses the validity of the `seedReveal` property of a block.
+This function assesses the validity of the `seedReveal` property of a block. `seedReveal` is a `SEED_LENGTH` bytes value with the seed revealed by the generator of a certain block.
```

Comment:
I would make it a bit more clear here, perhaps mentioning that similar to the `seedReveal` property of `blockHeaderAssetRandomModule` schema.

Reply:
Updated.